gulpfile.js

```diff
@@ -199,28 +193,22 @@ gulp.task('sass', function() {
 });
 
 // Builds the complete project from the sources into the target directory
-gulp.task('build', function(callback) {
-  // First run 'clean', then the other tasks
-  // TODO remove runSequence on Gulp 4.0 and use built in serial execution instead
-  runSequence('clean',
-    ['html', 'browserify', 'sass'],
-    callback);
-});
+gulp.task('build', gulp.series('clean', gulp.parallel('html', 'browserify', 'sass')));
 
-gulp.task('build-prod', function(callback) {
-  production = true;
-  runSequence('lint', 'build', callback);
-});
+gulp.task('build-prod', gulp.series(function(callback) {
+  production = true
+  callback();
```

Comment:
I think the callback is not necessary (no async code)

Reply:
I thought the same and I didn't have it in the first place. 

But if I change it to:
```ts
gulp.task('build-prod', gulp.series(function() {
  production = true;
}, 'lint', 'build'));
```
I get an error:
```
$ gulp build-prod
Using gulpfile ~/Projects/bitmovin-player-ui/gulpfile.js
Starting 'build-prod'...
Starting '<anonymous>'...
The following tasks did not complete: build-prod, <anonymous>
Did you forget to signal async completion?
```

Therefore, I added the callback and it works fine then. Happy to change it if we can find out what the problem is, but currently I don't really have an idea.

Reply:
Missed that part, thanks!